Apps/WebClient/src/Server/Services/IUserProfileService.cs

```diff
@@ -72,13 +73,13 @@ public interface IUserProfileService
         /// </summary>
         /// <param name="userPreference">The userPreference to create.</param>
         /// <returns>A userPreference wrapped in a RequestResult.</returns>
-        RequestResult<UserPreferenceModel> CreateUserPreference(UserPreferenceModel userPreference);
+        bool UpdateUserPreference(string hdid, string name, string value);
```

Comment:
Updates should only be run against objects as the version number is required.

Reply:
The delegate method does not have a create method either